core-services/prow/02_config/open-cluster-management-io/governance-policy-nucleus/_pluginconfig.yaml

```diff
@@ -0,0 +1,4 @@
+plugins:
+  open-cluster-management-io/governance-policy-nucleus:
+    plugins:
+    - dco
```

Comment:
Is there a reason this isn't just on https://github.com/openshift/release/blob/master/core-services/prow/02_config/open-cluster-management-io/_pluginconfig.yaml ?

Reply:
/shrug I'm not actually sure. For `stolostron`, they didn't want it enabled on private repos since it was slightly disruptive initially, but we're actually the only ones enabling it in Prow in the community. I figure if we're going to have Prow around, might as well enable all the things and not maintain it ourselves.

Reply:
Also, I'm very sad `/shrug` didn't work in the comments here. 🙂

Reply:
Yeah, seems like shrug should work: https://github.com/openshift/release/blob/master/core-services/prow/02_config/openshift/_pluginconfig.yaml#L52 - maybe it doesn't work in comment threads?

Anyway, I don't think we **need** to move it to that file, this is fine.

Reply:
/shruggy